packages/shared/src/components/modals/common.tsx

```diff
@@ -232,6 +232,13 @@ const GiftPlusModal = dynamic(
   () => import(/* webpackChunkName: "giftPlusModal" */ '../plus/GiftPlusModal'),
 );
 
+const GiftReceivedPlusModal = dynamic(
+  () =>
+    import(
+      /* webpackChunkName: "GiftReceivedPlusModal" */ '../plus/GiftReceivedPlusModal'
```

Comment:
Just to follow the convention.

```suggestion
      /* webpackChunkName: "giftReceivedPlusModal" */ '../plus/GiftReceivedPlusModal'
```